packages/frontend/src/pages/Login.tsx

```diff
@@ -123,41 +124,53 @@ const Login: FC = () => {
             {health.data?.auth.oneLogin.enabled && <OneLoginLoginButton />}
         </>
     );
+
     const passwordLogin = allowPasswordAuthentication && (
-        <Form name="login" methods={methods} onSubmit={handleLogin}>
-            <InputField
-                label="Email address"
-                name="email"
-                placeholder="Your email address"
-                disabled={isLoading}
-                rules={{
-                    required: 'Required field',
-                }}
-            />
-            <PasswordInputField
-                label="Password"
-                name="password"
-                placeholder="Your password"
-                disabled={isLoading}
-                rules={{
-                    required: 'Required field',
-                }}
-            />
-            <SubmitButton
-                type="submit"
-                intent={Intent.PRIMARY}
-                text="Sign in"
-                loading={isLoading}
-                data-cy="login-button"
-            />
-            {health.data?.hasEmailClient && (
-                <AnchorLinkWrapper>
-                    <AnchorLink href="/recover-password">
-                        Forgot your password ?
-                    </AnchorLink>
-                </AnchorLinkWrapper>
-            )}
-        </Form>
+        <form
+            name="login"
+            onSubmit={form.onSubmit((values: LoginParams) => mutate(values))}
+        >
+            <Stack spacing="lg">
+                <TextInput
+                    label="Email address"
+                    name="email"
+                    placeholder="Your email address"
+                    required
+                    {...form.getInputProps('email')}
+                    disabled={isLoading}
+                    styles={{
+                        label: {
+                            marginBottom: '5px',
+                        },
+                    }}
+                />
+                <PasswordInput
+                    label="Password"
+                    name="password"
+                    placeholder="Your password"
+                    required
+                    {...form.getInputProps('password')}
+                    disabled={isLoading}
+                    styles={{
+                        label: {
+                            marginBottom: '5px',
+                        },
```

Comment:
I am wondering if we can replace label margins as a global style in the MantineProvider

Reply:
not the global style for all <label> tags but for individual input elements

Reply:
will do in separate PR